src/Components/GitHubProfile/GitHubProfile.jsx

```diff
@@ -0,0 +1,105 @@
+// Apollo Imports
+import { useQuery, gql } from "@apollo/client";
+// MUI Imports
+import { Card, Typography, Stack } from "@mui/material";
+// Component Imports
+import CreateBanana from "../Banana/CreateBanana";
+// Store Imports
+import { useSelector, useDispatch } from "react-redux";
+
+
+const GitHubProfile = () => {
+    const dispatch = useDispatch();
+
+    // Banana Redux Store
+    const bananaStore = useSelector((state) => state.bananaReducer)
+
+    // Apollo GraphQl Query
+    const GITHUB_QUERY = gql `
+        {
+            user(login: "NH-pro") {
+            avatarUrl
+            bio
+            url
+            }
+        }
+    `;
+    const { loading, error, data } = useQuery(GITHUB_QUERY);
+    if (loading) return "Fetching my profile from GitHub!";
+    if (error) return <pre>{error.message}</pre>
+    
+
+    function picClickHandle(url) {
+        window.open(url, '_blank').focus();
+    }    
```

Comment:
This can be accomplished with [react-router](https://stackoverflow.com/questions/30202755/react-router-open-link-in-new-tab).  I tend to think if you are already using a package for something, like `react-router` for navigation that you should try to stick with that package for all adjacent activities.  But either way works.

Reply:
For example, I needed to do some date math in a work project a couple days ago.  The project has `moment` installed so I used that.  Then after work I watched a youtube video on how to easily do date math with a built in Node api called [intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl).  The next day I went into the project to refactor the `moment` work I did, but stopped because it didn't seem right to break away from the way my team had set as the standard for how we handle math involving dates.  On personal projects you obviously can work however you want, but try to pick a consistent method.